test/e2e/utils/fixture/fixture.go

```diff
@@ -8,6 +8,7 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	apiextension "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
```

Comment:
good improvement. Can squash this commit with the one(s) introducing the `fixture`?